test/specs/rules/require-capitalized-constructors-new.js

```diff
@@ -0,0 +1,54 @@
+var Checker = require('../../../lib/checker');
+var expect = require('chai').expect;
+
+describe('rules/require-capitalized-constructors-new', function() {
+    var checker;
+
+    function baseCases() {
```

Comment:
Means, that we could move those tests from the helper function and to the `'with `true` value'` `describe`